apps/alphabet_mobile/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This is not good. We do not want a separate package.json for a project in the monorepo. The point in the monorepo is that there's only one version of each dependency, shared by all apps. You need to look into how to setup the project with Nx. If that's not possible, we'll need to think carefully about whether this should just be a separate repo.